test/commands.js

```diff
@@ -766,6 +768,28 @@ async function runMVMWaitForMonthCommand(command, state) {
 
   return state;
 }
+
+async function runMVMPauseCommand(command, state) {
+  if (state.MVM !== undefined) {
+    let fromAccount = gen.getAccount(command.fromAccount),
+      shouldThrow = (state.MVMPaused == command.pause) || (command.fromAccount != state.owner);
+
+    try {
+      if (command.pause) {
+        await state.MVM.pause({from: fromAccount});
+      } else {
+        await state.MVM.unpause({from: fromAccount});
+      }
+
+      state.MVMPaused = command.pause;
```

Comment:
It will be useful to save the timestamp when it was paused, right? Maybe save that instead on a boolean, if it was paused it will be higher than 0. And we can compare the total time paused too, that is used on `getCurrentIndex()` if you try to get the current index after it was paused are you not receiving any error comparing to the expected one?

Reply:
Yes, you are right, it's important to keep track of that in the tests too, to verify it works well. I'll add it :+1: